src/components/Dropdown/DropdownItem.vue

```diff
@@ -29,7 +29,7 @@ export default {
       required: true
     },
     option: {
-      type: [String, Object],
+      type: [String, Object, undefined],
       required: true,
       validator: function (value) {
```

Comment:
Does the validator pass, or does it throw an error/warning in console? With undefined I guess it would return false.